pkg/volume/testing/testing.go

```diff
@@ -1281,7 +1280,7 @@ func FindEmptyDirectoryUsageOnTmpfs() (*resource.Quantity, error) {
 		used, err := resource.ParseQuantity("0")
 		return &used, err
 	}
-	tmpDir, err := utiltesting.MkTmpdir("metrics_du_test")
+	tmpDir, err := os.MkdirTemp("", "metrics_du_test")
 	if err != nil {
 		return nil, err
 	}
```

Comment:
```suggestion
        tmpDir := t.TempDir()
```

Reply:
this is not _test.go file, can't use t.TempDir() I think

Reply:
makes sense. Sorry, I thought you only change `_test.go` files.